src/averageAges.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+
+function calculateAverageAge(people) {
+  return people.reduce((acc, person) => (
+    acc + person.died - person.born
+  ), 0) / people.length;
```

Comment:
it's gonna be more readable if you create variable for people.reduce result and then return this variable / people.length;